lib/v2/xkb/index.js

```diff
@@ -0,0 +1,58 @@
+const got = require('@/utils/got');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+const UA = require('@/utils/rand-user-agent')({ browser: 'mobile safari', os: 'ios', device: 'mobile' });
+
+module.exports = async (ctx) => {
+    const channel = ctx.params.channel ?? 1;
+    const currentUrl = `https://www.xkb.com.cn/xkbapp/fundapi/article/api/articles?chnlId=${channel}&visibility=1&page=0&size=15&keyword=`;
+
+    const { data: response } = await got({
+        method: 'get',
+        url: currentUrl,
+        headers: {
+            siteId: '35',
+            'User-Agent': UA,
+        },
+    });
+
+    const list = response.data
+        .map((item) => ({
+            title: item.listTitle,
+            description: art(path.join(__dirname, 'templates/description.art'), {
+                thumb: item.metaInfo.thumbnails[0],
+            }),
+            pubDate: timezone(parseDate(item.operTime), +8),
+            link: 'https://www.xkb.com.cn/#/detail?id=' + item.id,
+            contentUrl: item.contentUrl,
+            author: item.metaInfo.author,
+            chnlName: item.metaInfo.chnlName,
+        }))
+        .filter((i) => i.contentUrl); // Remove "专题报道" (special report)
+
+    let chnlName = '';
+
+    const items = await Promise.all(
+        list.map((item) =>
+            ctx.cache.tryGet(item.contentUrl, async () => {
+                const detailResponse = await got({
+                    method: 'get',
+                    url: item.contentUrl,
+                });
+                const data = detailResponse.data.htmlContent.replace(/ \n/, '').replace(/\\/, '');
```

Comment:
@TonyRL Hi, sorry for bothering, how can I pass this check if I just want to replace only one backslash?

Reply:
What are you trying to replace? I think using `got().data` which involved a `JSON.parse()` will replace all those `\"` to `"`. Also, it is suggested to replace all `\n` with `<br>`.

Reply:
@TonyRL Yeah I want to replace `\"` to `"`, so I just want to remove those `\`. I'll try `JSON.parse()` later, but I'm still curious, if I just want to replace `\`, how can I do it so that code scanning won't throw an error?

Reply:
Using `got().data` already involved a `JSON.parse()`, you can't  do it again, you can check `lib/utils/got.js`. Beside that, you can't replace only one `\"` with `"`, you need to replace all occurrence.

Reply:
@TonyRL Oops, I got that, so I have to use `/\\/g`...... I misunderstood the meaning of that error. Thank you very much for answering my stupid mistake.